src/fiskaltrust.Launcher/Configuration/CashBoxConfigurationExt.cs

```diff
@@ -11,12 +12,9 @@ public static class CashBoxConfigurationExt
         private const string ENCRYPTION_SUFFIX = "_encrypted";
         private static readonly List<string> _configKeyToEncrypt = new() { "connectionstring" };
 
-        public static void Decrypt(this ftCashBoxConfiguration cashboxConfiguration, ECDiffieHellman clientEcdh, LauncherConfiguration launcherConfiguration)
+        public static void Decrypt(this ftCashBoxConfiguration cashboxConfiguration, LauncherConfiguration launcherConfiguration)
```

Comment:
I'm not sure why we changed this now 😅 the cashbox config is coming from the configuration service, which uses the shared secret (and we should absolutely stick to that, IMO). It's not related to locally encrypting the Launcher config, right? 

Can we move this back to the shared secret logic?

Reply:
@volllly or what were the reasons for changing this? s😅

Reply:
The shared secret is now computed once in the encryption helper directly and stored in a static... should we pull that out for better clarity?

Reply:
@TSchmiedlechner I pulled the secret out of the Helper 😁 I think its clearer now

Reply:
Thanks 😁 I was confused by the fact that we didn't store the key, but you're completely right, this was also missing before. 
We'll add this in a separate PR, I guess :)